xwiki-rendering-api/src/main/java/org/xwiki/rendering/listener/MetaData.java

```diff
@@ -57,6 +57,13 @@
      */
     public static final String BASE = "base";
 
+    /**
+     * Represents an inline editable content.
+     *
+     * @since 10.9RC1
+     */
+    public static final String INLINE_EDITABLE = "wysiwyg-editable-content";
```

Comment:
Since you change the class name you should also change this into something less WYSIWYG specific.